ipmi-config/ipmi-config-category-core-lan6-conf-section.c

```diff
@@ -137,12 +137,15 @@ _get_ipv6_ipv4_support (ipmi_config_state_data_t *state_data,
 }
 
 static ipmi_config_err_t
-ipv6_ipv4_support_commit (ipmi_config_state_data_t *state_data,
-                          const char *section_name,
-                          const struct ipmi_config_keyvalue *kv)
+read_only_commit (ipmi_config_state_data_t *state_data,
```

Comment:
i like the idea, but I think here's a better idea:

Use the flag IPMI_CONFIG_CHECKOUT_KEY_COMMENTED_OUT and IPMI_CONFIG_READABLE_ONLY when calling ipmi_config_section_add_key() below (I had forgotten READABLE_ONLY already existed, sorry, could have mentioned earlier).

Then lets put this funciton into ipmi-config-utils.[ch], as a convenience function any read only values can use.